### PR TITLE
Create Dockerfile for ETL pipeline

### PR DESCRIPTION
**Pull request by qrafiq1**
Lambda handler has been made and Dockerfile, the handler has a general exception to catch all errors just incase.

### CODE REVIEW

File: pipeline/etl.py [hunk 1]
```diff
@@ -13,6 +13,22 @@
...
 
+def lambda_handler(event, context):
+    """Runs the ETL pipeline when the lambda is invoked"""
+    try:
+        main()
+        return {
+            "statuscode": 200,
+            "body": "ETL pipeline executed successfully!"
+        }
+    except Exception as e:
```
Comment (FarihaChoudhury) [inline pipeline/etl.py:24]: Could we be able to be more specific with the kind of exception here?

Comment (FarihaChoudhury) [reply pipeline/etl.py:24]: If there's many different kinds of exceptions that could occur here, should we pylint disable it?

Comment (qrafiq1) [reply pipeline/etl.py:24]: I will pylint disable it, having a general exception would be better as there are errors we may not be able to catch with a specfic exception.

**Review by SurinaCS — APPROVED** [review]
great work, looks really clean

**Pull request merged**